### PR TITLE
xdg-ninja: init at 0.2.0.1

### PR DESCRIPTION
**Pull request by arcuru**
###### Description of changes

xdg-ninja is a script that checks your $HOME directory for unwanted files and directories, and provides advice for how to force programs to respect the XDG Base Directory Specification. See https://github.com/b3nj5m1n/xdg-ninja

It is a POSIX script that expects the 'programs/' subdirectory to exist in the same directory as the script.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/xdg-ninja/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  '';
+
+  meta = with lib; {
+    description =
+      "A shell script which checks your $HOME for unwanted files and directories.";
+    homepage = "https://github.com/b3nj5m1n/xdg-ninja";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ patricksjackson ];
+    mainProgram = "xdg-ninja";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/xdg-ninja/default.nix:36]:
```suggestion
```
Thats the default

File: pkgs/tools/misc/xdg-ninja/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ln -s "$out/share/xdg-ninja/xdg-ninja.sh" "$out/bin/xdg-ninja"
+
+    wrapProgram "$out/bin/xdg-ninja" \
+      --prefix PATH : "${lib.makeBinPath [ glow jq ]}"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description =
+      "A shell script which checks your $HOME for unwanted files and directories.";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/xdg-ninja/default.nix:31]:
```suggestion
    description = "A shell script which checks your $HOME for unwanted files and directories.";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/xdg-ninja/default.nix (hunk 2 above)
Comment (OPNA2608) [reply pkgs/tools/misc/xdg-ninja/default.nix:31]: https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
> Don’t include a period at the end.

```suggestion
    description = "A shell script which checks your $HOME for unwanted files and directories";
```

File: pkgs/tools/misc/xdg-ninja/default.nix (hunk 1 above)
Comment (patricksjackson) [reply pkgs/tools/misc/xdg-ninja/default.nix:36]: Good to know, thanks!

**Comment by patricksjackson** [comment]
Thanks! Updated with requested changes.

**Review by ck3d — APPROVED** [review]
LGTM

**Pull request merged**